### PR TITLE
Upload reason failure

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Fixes https://github.com/ooni/probe/issues/919
This has been tested “thanks” to this bug https://github.com/ooni/probe/issues/899

### CODE REVIEW

File: ooniprobe/View/TestResults/LogViewController.m [hunk 1]
```diff
@@ -18,67 +18,87 @@ - (void)viewDidLoad {
...
+    NSString *fileName = [self.measurement getLogFile];
+    self.text = [TestUtility getUTF8FileContent:fileName];
+    if (self.text != nil) {
+        dispatch_async(dispatch_get_main_queue(), ^{
+            [self.textView setText:self.text];
+        });
+    }
+    else {
+        //Show Log not found alert, go back on OK.
+        //This will be useful for when we'll implement the auto log deletion
```
Comment (bassosimone) [inline ooniprobe/View/TestResults/LogViewController.m]: Didn't we implement this already?

Comment (lorenzoPrimi) [reply ooniprobe/View/TestResults/LogViewController.m]: I just moved it in a function to remove it from the big if in the main function

Comment (bassosimone) [reply ooniprobe/View/TestResults/LogViewController.m]: Didn't we implement auto-log deletion?

Comment (lorenzoPrimi) [reply ooniprobe/View/TestResults/LogViewController.m]: Oh you refer to the second comment, I guess that can be deleted

**Pull request merged**